ui/src/test/SortCursor.test.js

```diff
@@ -1,45 +1,19 @@
-import { fireEvent, render, screen } from "@testing-library/react";
+import {render, screen } from "@testing-library/react";
 import SortCursor from "../Components/Dashboard/Table/TableHeader/SortCursor";
 import { TABLE } from "../Constant/constant";
 
 describe("SortCursor Component", () => {
+  it("Should render the SortCursor Component, which includes Ascending cursor", () => {
+    render(<SortCursor />);
 
-    it("Should render the SortCursor Component with Ascending cursor", () => {
-      render(<SortCursor />)
+    const cursor = screen.getByTitle(TABLE.ASCENDING);
+    expect(cursor).toBeInTheDocument();
+  });
 
-      const cursor = screen.getByTitle(TABLE.ASCENDING)
-      expect(cursor).toBeInTheDocument(); 
-    })
-
-    it("Should render the SortCursor Component with Descending cursor", () => {
-      render(<SortCursor />)
-
-      const cursor = screen.getByTitle(TABLE.DESCENDING)
-      expect(cursor).toBeInTheDocument();
-       
-    })
-
-    it("Should fire event on click ascending", async () => {
```

Comment:
Why test related to fireEvent removed?

Reply:
Hi @DeRaowl we moved this click event from the cursor to the header. So we have to implement tests for the header